src/scripts/user/frontPage.jsx

```diff
@@ -0,0 +1,122 @@
+// dependencies -------------------------------------------------------
+
+import React           from 'react';
+import Reflux          from 'reflux';
+import userStore       from './user.store.js';
+import {Link}          from 'react-router';
+import Spinner         from '../common/partials/spinner.jsx';
+import {Tabs, Tab}     from 'react-bootstrap';
+import Footer          from '../common/partials/footer.jsx';
+import FrontPageTabs   from '../common/partials/frontPageTabs.jsx';
+import BrowsePipelines from '../common/partials/browsePipelines.jsx';
+
+// component setup ----------------------------------------------------
+
+let FrontPage = React.createClass({
+
+    mixins: [Reflux.connect(userStore)],
+
+// life cycle events --------------------------------------------------
+
+    render () {
+        let form;
```

Comment:
I would move each variable between here and 108 into individual template functions to keep the render method clean.